Makefile

```diff
@@ -131,19 +150,26 @@ docker-push:
 #####################################################################
 
 .PHONY: install-requirements
-install-requirements:
+install-requirements: vendor-requirements install-extra-requirements
+
+.PHONY: vendor-requirements
+vendor-requirements:
 	@go install -mod=vendor github.com/onsi/ginkgo/ginkgo
 	@go install -mod=vendor github.com/ahmetb/gen-crd-api-reference-docs
 	@go install -mod=vendor github.com/golang/mock/mockgen
 	@go install -mod=vendor sigs.k8s.io/controller-tools/cmd/controller-gen
 	@GO111MODULE=off go get github.com/prometheus/prometheus/cmd/promtool
+
+.PHONY: install-extra-requirements
+install-extra-requirements:
 	@./hack/install-requirements.sh
 
 .PHONY: revendor
 revendor:
 	@GO111MODULE=on go mod vendor
 	@GO111MODULE=on go mod tidy
 	@curl -sSLo hack/setup-envtest.sh https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/$(CR_VERSION)/hack/setup-envtest.sh
+	@cd vendor/k8s.io/code-generator; go mod vendor
```

Comment:
/invite @ialidzhikov - do you have opinions on this change?

Reply:
The same question is already asked above but I still don't get why this is required. We are using `./vendor/k8s.io/code-generator/generate-groups.sh` and `./vendor/k8s.io/code-generator/generate-internal-groups.sh` scripts and I don't see why such change is required.
@poelzi , if you believe this is something specific to your env, can you try to reproduce with https://github.com/kubernetes/sample-apiserver locally and open issue to https://github.com/kubernetes/kubernetes?